test/unit_tests/arkode/CXX_serial/ark_test_dahlquist_mri.cpp

```diff
@@ -418,10 +414,11 @@ int run_tests(MRISTEP_METHOD_TYPE type, sunrealtype t0, int nsteps,
 
     cout << "\nComparing Solver Statistics:\n";
 
+    int nstages_evaluated = nstages_stored - stiffly_accurate;
```

Comment:
Since `stiffly_accurate` is a bool, I think the following might be better. 
```suggestion
    int nstages_evaluated = nstages_stored;
    if (stiffly_accurate) nstages_evaluated--;
```